Feature/global ens metplus5

BinbinZhou-NOAA opened the pull request:
## Pull Request Testing ##

- [y ] Describe testing already performed for this Pull Request:</br>

- [ y] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- [ y] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [y ] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [y ] Please complete this pull request review by **[Fill in date]**.</br>

## Pull Request Checklist ##

- [ y] Review the source issue metadata (required labels, projects, and milestone).
- [y ] Complete the PR description above.
- [y ] Ensure the PR title matches the feature branch name.
- [y ] Check the following:
- [y ]  Instructions provided on how to run
- [y ]  Developer's name is replaced by ${user} where necessary throughout the code
- [y ]  Check that the ecf file has all the proper definitions of variables
- [y ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [y ]  Check to see that the output directory structure is followed
- [y ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ y] After submitting the PR, select **Development** issue with the original issue number.
- [ y] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [y ] Close the linked issue.

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/global_ens_metplus5

BinbinZhou-NOAA committed:
First update

BinbinZhou-NOAA committed:
update run.ver

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/global_ens_metplus5

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/global_ens_metplus5

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/global_ens_metplus5

BinbinZhou-NOAA committed:
delete naefs7 test ecf

BinbinZhou-NOAA committed:
Update

BinbinZhou-NOAA committed:
delete naefs7 ecf

BinbinZhou-NOAA committed:
delete __pycache__

BinbinZhou-NOAA committed:
new update

BinbinZhou-NOAA committed:
delete naefs7 ecf

BinbinZhou-NOAA committed:
update ecf files and scripts files

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/global_ens_metplus5

BinbinZhou-NOAA committed:
update ecf files

BinbinZhou-NOAA committed:
update ecf and J-job

BinbinZhou-NOAA committed:
update plots ecf files

BinbinZhou-NOAA commented:
The prep,  stat and plots testing instruction can be found 
https://docs.google.com/document/d/1Ij5y-sFDMJguAv3L_x8cjFmw5LraBkx7xH_b2--gZU4/edit

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/global_ens_metplus5

BinbinZhou-NOAA committed:
add checking/notifying missing data

BinbinZhou-NOAA committed:
Bug fix/update plot ecf files

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/global_ens_metplus5

BinbinZhou-NOAA committed:
Change run.ver to run.ver.metplus5.0.0

BinbinZhou-NOAA committed:
Change prepbufr directory from gfs to gdas path

BinbinZhou-NOAA committed:
Update to remove sh in execution of scripts

BinbinZhou-NOAA committed:
update  ush/global_ens/global_det_atmos_util.py to remove missing fields in osi data

BinbinZhou-NOAA committed:
update naefs plot ecf files

BinbinZhou-NOAA commented:
Perry,

  For your question put in href PR but for global_en:
"Somehow, there were a lot of changes in the global_ens section of the code when you made your update. Please remove ALL global_ens changes from this particular feature branch, and try to reinstall the authoritative branch codes. We do not want this particular PR to mess with the global_ens in the authoritative branch"  

 Yes,  since the global PR was submitted,  there are several changes that MUST to be addressed for global_ens, particularly
1) change run.ver to run.ver.metplus5.0.0 (must be changed) 
2) jobid added to ecf file (Logan suggested) 
3) prepbufr from gfs to gdas path (Geoff M. suggested)
4) removed "sh" inthe script execution  statements (Logan suggested)
5) Missing data email alert 
6) Removed missing fields in the osi_saf python scripts.

1) and 6) are must be changed, otherwise the stats package will fail to run. 

If the current PR can not be tested, let me close the global_ens PR, then re-open it when you are ready to test?
Please give me your further advice.

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/global_ens_metplus5

BinbinZhou-NOAA committed:
update naefs plot and headline file names

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/global_ens_metplus5

BinbinZhou-NOAA committed:
update naefs grid2grid conf files

BinbinZhou-NOAA committed:
update GridStat_fcstNAEFSbc_obsModelAnalysis_climoERA5_mean.conf

BinbinZhou-NOAA committed:
update GenEnsProd_fcstNAEFSbc_obsModelAnalysis_climoERA5.con

BinbinZhou-NOAA committed:
correct file names Pb2nc_obsGFS_Prepbufr.*.cong to Pb2nc_obsGFS_Prepbufr.*.conf

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/global_ens_metplus5

BinbinZhou-NOAA committed:
Update py scripts by adding top head

BinbinZhou-NOAA committed:
add top head to py scripts

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/global_ens_metplus5

BinbinZhou-NOAA committed:
update headline plot codes

BinbinZhou-NOAA committed:
update ush/global_ens/evs_global_ens_headline_plot.sh

PerryShafran-NOAA commented:
I will be starting this work now.  In the meantime, please be sure to merge the current develop branch into this feature branch.

Thanks!

Perry

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/global_ens_metplus5

PerryShafran-NOAA commented:
Question for you:  does the jevs_global_ens_headline_prep.ecf need to be submitted after the jevs_global_ens_atmos_prep.ecf file is complete?  I ran them both together and it seems that the headline prep file failed because it couldn't find items in the prep directory. 

I'm going to try to resubmit the headline prep job after the global_ens_atmos prep job is complete.

Perry

BinbinZhou-NOAA commented:
Yes, submit  jevs_global_ens_headline_prep.ecf after
jevs_global_ens_atmos_prep.ecf is completed.

Binbin


On Fri, May 19, 2023 at 9:24 AM PerryShafran-NOAA ***@***.***>
wrote:

> Question for you: does the jevs_global_ens_headline_prep.ecf need to be
> submitted after the jevs_global_ens_atmos_prep.ecf file is complete? I ran
> them both together and it seems that the headline prep file failed because
> it couldn't find items in the prep directory.
>
> I'm going to try to resubmit the headline prep job after the
> global_ens_atmos prep job is complete.
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/93#issuecomment-1554573052>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFPPUF6FPYZAWRXA2VTXG5X7TANCNFSM6AAAAAAWVWZC7Y>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

PerryShafran-NOAA commented:
The prep step has been completed:

Please look at the results here:  /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/prep/global_ens/atmos.20230517 and headline.20230517.  

I ran three prep steps:  atmos, naefs, and headline.  Please see the .o files here: /lfs/h2/emc/vpppg/noscrub/perry.shafran/EVS/ecf/global_ens/prep.

While we're looking at these files, please change the names of these files to:

jevs_global_ens_headline_prep.oNNNNN
jevs_global_ens_atmos_prep.oNNNNN
jevs_global_ens_naefs_prep.oNNNN

Also, if you look in the directory /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/prep/global_ens, you'll see that there are directories called atmos and headline (no dates) being created.  Can you look into this?  We should not be making these directories.

Perry

BinbinZhou-NOAA commented:
Perry,

  I checked and compared your output files, all are fine except one file
ccpa.t12z.grid3.24h.f00.nc is missing in
 /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/prep/global_ens/atmos.20230517/gefs
sub-directory
The reason is this ccpa is 24h accumulation that requires ccpa 6h
accumulation fire at 18Z on previous day.
But you just run one day (20230517) and there is no 20230515 prep files, so
this 24h ccap can not be
generated. If you run tomorrow (20230518), then this file can be generated
since you have prep data on 20230517.
  Anyway, you still can go ahead to test stat ecf jobs

Binbin



On Fri, May 19, 2023 at 1:12 PM PerryShafran-NOAA ***@***.***>
wrote:

> The prep step has been completed:
>
> Please look at the results here:
> /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/prep/global_ens/atmos.20230517
> and headline.20230517.
>
> I ran three prep steps: atmos, naefs, and headline. Please see the .o
> files here: /lfs/h2/emc/vpppg/noscrub/perry.shafran/EVS/ecf/global_ens/prep.
>
> While we're looking at these files, please change the names of these files
> to:
>
> jevs_global_ens_headline_prep.oNNNNN
> jevs_global_ens_atmos_prep.oNNNNN
> jevs_global_ens_naefs_prep.oNNNN
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/93#issuecomment-1554977786>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFKOZOWAD5DBEG6Q7A3XG6SWVANCNFSM6AAAAAAWVWZC7Y>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

PerryShafran-NOAA commented:
Thank you for that info.  Do you have data that I can put in my test directories so I can better go ahead and move on to stats?  

Since it's late on Friday, I'll bring your data in on Monday and then go to the stats step.

Thanks!

Perry

BinbinZhou-NOAA commented:
Yes,

  Copy the file  ccpa.t12z.grid3.24h.f00.nc
from /lfs/h2/emc/vpppg/noscrub/binbin.zhou/evs/v1.0/prep/global_ens/atmos.20230517/gefs
to your
/lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/prep/global_ens/atmos.20230517

  Please note, right now you  have the prep data files only on one day. To
verify all 16 days of  global_ens forecast,
you must have 16days of prep data on your account.  Alternatively, you can
use my prep data in directory
   /lfs/h2/emc/vpppg/noscrub/binbin.zhou/evs/v1.0/prep/global_ens

Binbin

On Fri, May 19, 2023 at 3:36 PM PerryShafran-NOAA ***@***.***>
wrote:

> Thank you for that info. Do you have data that I can put in my test
> directories so I can better go ahead and move on to stats?
>
> Since it's late on Friday, I'll bring your data in on Monday and then go
> to the stats step.
>
> Thanks!
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/93#issuecomment-1555142243>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFJUJAJ6C5V6SDAURLLXG7DUXANCNFSM6AAAAAAWVWZC7Y>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/global_ens_metplus5

PerryShafran-NOAA commented:
Hi, Binbin,

I pulled in all the prep data for May into my prep directory and ran all the stats jobs.  Would you please take a look under the directory /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/stats/global_ens and see if all looks good?

I don't see any headline final stat files, though, but there is a check I can do to see why those aren't working.  Let me take a look, but please look at all the rest of the stat files.

Thanks!

Perry

BinbinZhou-NOAA commented:
OK, I'll check them. It seems login to cactus is very slow right now.

Binbin

On Mon, May 22, 2023 at 1:39 PM PerryShafran-NOAA ***@***.***>
wrote:

> Hi, Binbin,
>
> I pulled in all the prep data for May into my prep directory and ran all
> the stats jobs. Would you please take a look under the directory
> /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/stats/global_ens and see
> if all looks good?
>
> I don't see any headline final stat files, though, but there is a check I
> can do to see why those aren't working. Let me take a look, but please look
> at all the rest of the stat files.
>
> Thanks!
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/93#issuecomment-1557634898>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFIYRDAYDTDGEH5KR5DXHOQF3ANCNFSM6AAAAAAWVWZC7Y>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA commented:
Perry,

  I have checked all output stat files, all are good except for headline
stat files.
Actually my headline stat files were also not generated today (all were
created but today).
So let me first figure out the reason.

Binbin

On Mon, May 22, 2023 at 1:39 PM PerryShafran-NOAA ***@***.***>
wrote:

> Hi, Binbin,
>
> I pulled in all the prep data for May into my prep directory and ran all
> the stats jobs. Would you please take a look under the directory
> /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/stats/global_ens and see
> if all looks good?
>
> I don't see any headline final stat files, though, but there is a check I
> can do to see why those aren't working. Let me take a look, but please look
> at all the rest of the stat files.
>
> Thanks!
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/93#issuecomment-1557634898>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFIYRDAYDTDGEH5KR5DXHOQF3ANCNFSM6AAAAAAWVWZC7Y>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

PerryShafran-NOAA commented:
You don't have a prep directory for the headline scores for 20230520.  

I'm running the headline stats at 20230519 to see if they are ok.  There is a headline prep directory for that date.

Perry

PerryShafran-NOAA commented:
The headline job ran for 20230519 - there is a file here:  /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/stats/global_ens

Let me also run for GFS and NAEFS and see if I can get a file.

Perry

BinbinZhou-NOAA commented:
Perry,

   The headline.20230520 is not in my prep directory. I checked the output
of headline ecf and found
this message:
  mkdir: cannot create directory
â/lfs/h2/emc/ptmp/binbin.zhou/evs/tmpnwprd/jevs_headline_prep.58538031.cbqs01â:
Disk quota exceeded
So the reason is ptmp had no space. I'll rerun the headline prep ecf to get
headline prep data

Binbin

On Mon, May 22, 2023 at 2:16 PM PerryShafran-NOAA ***@***.***>
wrote:

> You don't have a prep directory for the headline scores for 20230520.
>
> I'm running the headline stats at 20230519 to see if they are ok. There is
> a headline prep directory for that date.
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/93#issuecomment-1557683344>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFMCHY7R6HCZ5MQT4F3XHOUOJANCNFSM6AAAAAAWVWZC7Y>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

PerryShafran-NOAA commented:
Thanks for letting us know - you may want to put your working directory in /stmp rather than /ptmp to prevent this.  Since we are putting all plot output into /ptmp, that disk can fill up.  We need to have the working directories in /stmp anyway.

Thanks!

Perry

BinbinZhou-NOAA commented:
Perry,

   The headline.20230502 has been created in my prep directory
/lfs/h2/emc/vpppg/noscrub/binbin.zhou/evs/v1.0/prep/global_ens
You can copy it and run the headline stat ecf for gefs, naefs and gfs.
I have run these 3 stat ecf jobs and all stats were created for the
headline


Binbin


On Mon, May 22, 2023 at 3:29 PM Binbin Zhou - NOAA Affiliate <
***@***.***> wrote:

> Perry,
>
>    The headline.20230520 is not in my prep directory. I checked the output
> of headline ecf and found
> this message:
>   mkdir: cannot create directory
> â/lfs/h2/emc/ptmp/binbin.zhou/evs/tmpnwprd/jevs_headline_prep.58538031.cbqs01â:
> Disk quota exceeded
> So the reason is ptmp had no space. I'll rerun the headline prep ecf to
> get headline prep data
>
> Binbin
>
> On Mon, May 22, 2023 at 2:16 PM PerryShafran-NOAA <
> ***@***.***> wrote:
>
>> You don't have a prep directory for the headline scores for 20230520.
>>
>> I'm running the headline stats at 20230519 to see if they are ok. There
>> is a headline prep directory for that date.
>>
>> Perry
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/NOAA-EMC/EVS/pull/93#issuecomment-1557683344>, or
>> unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AQBMPFMCHY7R6HCZ5MQT4F3XHOUOJANCNFSM6AAAAAAWVWZC7Y>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>
>
> --
>
> Binbin Zhou
>
> Physical Scientist
>
> Lynker at NOAA/NWS/NCEP/EMC
>
> 5830 University Research Ct.
>
> College Park, MD 20740
>
> ***@***.***
>
> 301-683-3683
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA commented:
Perry,

  Thanks for this good suggestion!

Binbin

On Mon, May 22, 2023 at 3:40 PM PerryShafran-NOAA ***@***.***>
wrote:

> Thanks for letting us know - you may want to put your working directory in
> /stmp rather than /ptmp to prevent this. Since we are putting all plot
> output into /ptmp, that disk can fill up. We need to have the working
> directories in /stmp anyway.
>
> Thanks!
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/93#issuecomment-1557839554>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFIQOJBXJ6W6KU6YSJDXHO6KFANCNFSM6AAAAAAWVWZC7Y>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/global_ens_metplus5

BinbinZhou-NOAA commented:
Perry,

  I checked your global_ens stat output files in atmos.20230520, all are
good.
If you want to run plots jobs for the past 31daya or 90days, one day stats
data are not enough.
But you can use my stats files
in  /lfs/h2/emc/vpppg/noscrub/binbin.zhou/evs/v1.0/stats/global_ens

Binbin

On Mon, May 22, 2023 at 4:16 PM Binbin Zhou - NOAA Affiliate <
***@***.***> wrote:

> Perry,
>
>   Thanks for this good suggestion!
>
> Binbin
>
> On Mon, May 22, 2023 at 3:40 PM PerryShafran-NOAA <
> ***@***.***> wrote:
>
>> Thanks for letting us know - you may want to put your working directory
>> in /stmp rather than /ptmp to prevent this. Since we are putting all plot
>> output into /ptmp, that disk can fill up. We need to have the working
>> directories in /stmp anyway.
>>
>> Thanks!
>>
>> Perry
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/NOAA-EMC/EVS/pull/93#issuecomment-1557839554>, or
>> unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AQBMPFIQOJBXJ6W6KU6YSJDXHO6KFANCNFSM6AAAAAAWVWZC7Y>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>
>
> --
>
> Binbin Zhou
>
> Physical Scientist
>
> Lynker at NOAA/NWS/NCEP/EMC
>
> 5830 University Research Ct.
>
> College Park, MD 20740
>
> ***@***.***
>
> 301-683-3683
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

PerryShafran-NOAA commented:
I will be testing the plots now.  I am linking to your stats archive to do this.  I will let you know when all the plot scripts are done.

Perry

PerryShafran-NOAA commented:
Can you explain the init_separate_plots scripts?  Do I need to run those as well as those without the init_separate_plots (for the gefs_grid2obs plot scripts)?

Thanks!

Perry

PerryShafran-NOAA commented:
All the plot jobs are complete.  The directories are here:

/lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/global_ens/atmos.20230521

The spatial map file seems to have no plots in it, and plus the file name does not have the proper filename.  

Headline here:  /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/global_ens/headline.20230521

Thanks!

Perry

BinbinZhou-NOAA commented:
Perry.

  The init_separate_plots scripts are for forecast of different initial
times
  I checked your  plot files, all are fine except for the spatial plot.
  Actually my spatial plot was also not generated today and yesterday, but
  it  was before yesterday. So I'll double check the reason.

Binbin




On Tue, May 23, 2023 at 2:48 PM PerryShafran-NOAA ***@***.***>
wrote:

> All the plot jobs are complete. The directories are here:
>
>
> /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/global_ens/atmos.20230521
>
> The spatial map file seems to have no plots in it, and plus the file name
> does not have the proper filename.
>
> Headline here:
> /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/global_ens/headline.20230521
>
> Thanks!
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/93#issuecomment-1559965366>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFJ5KTUHG2PX27A4PJDXHUBAHANCNFSM6AAAAAAWVWZC7Y>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA committed:
Update gefs spatial script file

BinbinZhou-NOAA commented:
Perry,

    The reason for failure of spatial plot was identified and fixed:
The problem is in GEFS's APCP24 ensemble  mean netcdf files. These files
are generated from MET's
GenEnsProd tool.  But on May 21, some of these netcdf files only have the
header  but no data body.
This only happened on May21.   I remember on that day I got "space
exceeding limit" issue  on ptmp.
After I changed DATA from ptmp to stmp,  this issue was resolved.
   So now you can re-run the global_ens spatial plot job to test.
 To avoid this issue in the future, I have modified/updated  the spatial
plot script.

Binbin



On Tue, May 23, 2023 at 3:34 PM Binbin Zhou - NOAA Affiliate <
***@***.***> wrote:

> Perry.
>
>   The init_separate_plots scripts are for forecast of different initial
> times
>   I checked your  plot files, all are fine except for the spatial plot.
>   Actually my spatial plot was also not generated today and yesterday, but
>   it  was before yesterday. So I'll double check the reason.
>
> Binbin
>
>
>
>
> On Tue, May 23, 2023 at 2:48 PM PerryShafran-NOAA <
> ***@***.***> wrote:
>
>> All the plot jobs are complete. The directories are here:
>>
>>
>> /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/global_ens/atmos.20230521
>>
>> The spatial map file seems to have no plots in it, and plus the file name
>> does not have the proper filename.
>>
>> Headline here:
>> /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/global_ens/headline.20230521
>>
>> Thanks!
>>
>> Perry
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/NOAA-EMC/EVS/pull/93#issuecomment-1559965366>, or
>> unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AQBMPFJ5KTUHG2PX27A4PJDXHUBAHANCNFSM6AAAAAAWVWZC7Y>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>
>
> --
>
> Binbin Zhou
>
> Physical Scientist
>
> Lynker at NOAA/NWS/NCEP/EMC
>
> 5830 University Research Ct.
>
> College Park, MD 20740
>
> ***@***.***
>
> 301-683-3683
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/global_ens_metplus5

PerryShafran-NOAA commented:
Thank you for that check.  Hopefully we have resolved the space issues on ptmp, but DATA should be going to stmp anyway.  

I'll re-run this particular plot, again pointing to your stats archive.

Perry

BinbinZhou-NOAA commented:
Perry,

   All of jobs' DATA have been changed to stmp.
  My global_ens stat directory is
  /lfs/h2/emc/vpppg/noscrub/binbin.zhou/evs/v1.0/stats/global_ens
  Hope the global_ens PR test can be finished today since tomorrow Cactus
may be down and
I'll take leave on Friday.

Binbin

On Wed, May 24, 2023 at 10:53 AM PerryShafran-NOAA ***@***.***>
wrote:

> Thank you for that check. Hopefully we have resolved the space issues on
> ptmp, but DATA should be going to stmp anyway.
>
> I'll re-run this particular plot, again pointing to your stats archive.
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/93#issuecomment-1561312184>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFKC4QYFGTUTTIBFD5LXHYOH7ANCNFSM6AAAAAAWVWZC7Y>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

PerryShafran-NOAA commented:
Hi, Binbin,

I see a larger tar file for the spatial plot file.  Please examine all the plot files here:  /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/global_ens/atmos.20230522

and the headline plots here:  /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/global_ens/headline.20230522

Thanks!

Perry

BinbinZhou-NOAA commented:
Perry,

   Yes, the tar file for spatial map in
/lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/global_ens/atmos.20230522
is good
and headline plot file in
 /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/global_ens/headline.20230522
is
also good

Binbin

On Wed, May 24, 2023 at 1:27 PM PerryShafran-NOAA ***@***.***>
wrote:

> Hi, Binbin,
>
> I see a larger tar file for the spatial plot file. Please examine all the
> plot files here:
> /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/global_ens/atmos.20230522
>
> and the headline plots here:
> /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/global_ens/headline.20230522
>
> Thanks!
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/93#issuecomment-1561663156>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFKDB3KORUJPH7T7ZLTXHZAGJANCNFSM6AAAAAAWVWZC7Y>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

PerryShafran-NOAA commented:
Sounds like you're saying that all is good.  We've checked prep, stats, and plots, and you have validated all of it.  I think we can merge this PR finally.

Perry

BinbinZhou-NOAA commented:
Yes, all are good!

Thanks!

Binbin

On Wed, May 24, 2023 at 1:39 PM PerryShafran-NOAA ***@***.***>
wrote:

> Sounds like you're saying that all is good. We've checked prep, stats, and
> plots, and you have validated all of it. I think we can merge this PR
> finally.
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/93#issuecomment-1561679665>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFLMFSDKJQ5LGEFXZ3LXHZBUVANCNFSM6AAAAAAWVWZC7Y>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683
